run-payinv.sh

```diff
@@ -3,6 +3,6 @@
 python src/manage.py migrate
 python src/manage.py check --deploy
 python src/manage.py collectstatic --no-input
-python src/manage.py compilemessages
+python src/manage.py compilemessages -l es
```

Comment:
Well, actually only have the messages for `es_ES`. You  should change by this key instead only `es`

Reply:
resolved in 7fc2160c2c9bbb9568a659ff3edf2526142d33fc

Reply:
Good!